src/radio/radio-group.component.ts

```diff
@@ -74,6 +76,17 @@ export class RadioGroup implements AfterContentInit, AfterViewInit, ControlValue
 
 	@Input() labelPlacement: "right" | "left" =  "right";
 
+	/**
+	 * Used to set the `aria-label` attribute on the input element.
+	 */
+	// tslint:disable-next-line:no-input-rename
+	@Input("aria-label") ariaLabel = "";
```

Comment:
The input should actually be `ariaLabel` - that way we don't generate extra labels unnessecarily (and it follows our existing convention) ... same for `labelledby`